pagination/offsetpagination/paginator.go

```diff
@@ -0,0 +1,79 @@
+package offsetpagination
+
+import (
+	"net/http"
+	"strconv"
+)
+
+const (
+	defaultPageSize int = 100
```

Comment:
I personally would reduce the default page sitze to 50.
Auth0 does this.

![image](https://github.com/user-attachments/assets/8330eb9d-3d6e-4dba-8ebe-595a46d6060b)